src/Pact/Native.hs

```diff
@@ -221,18 +221,18 @@ langDefs =
     ,defRNative "make-list" makeList (funType (TyList a) [("length",tTyInteger),("value",a)])
      "Create list by repeating VALUE LENGTH times. `(make-list 5 true)`"
 
-    ,defRNative "reverse" reverse' (funType (TyList a) [("l",TyList a)])
+    ,defRNative "reverse" reverse' (funType (TyList a) [("list",TyList a)])
      "Reverse a list. `(reverse [1 2 3])`"
```

Comment:
If you're bothering to give a nice variable name, the comment should be "Reverse LIST."